.github/workflows/tav-command.yml

```diff
@@ -72,15 +72,38 @@ jobs:
               }
               if (match[1]) {
                 if (match[1] !== 'all') {
-                  modules = match[1].split(',')
+                  modules = match[1].split(',').map((name) => modules.find((m) => m.name === name))
                 }
               }
               if (match[3]) {
-                versions = match[3].split(',')
+                versions = match[3].split(',').map(Number)
               }
             }
-            core.setOutput('modules', modules)
-            core.setOutput('versions', versions)
+
+            const badModuleIndex = modules.findIndex((m) => !m)
+            const badVersionIndex = versions.findIndex((v) => isNaN(v))
+
+            if (badModuleIndex !== -1) {
+              core.setFailed(`Incorrect modules list, please review it'`)
+              return
+            }
+            if (badVersionIndex !== -1) {
+              core.setFailed(`Incorrect versions list, please review it'`)
+              return
+            }
+
+            const permutations = []
+            for (const mod of modules) {
+              for (const nv of versions) {
+                if (mod.minVersion && nv >= mod.minVersion) {
+                  permutations.push(`${mod.name} ${nv}`)
+                } else if (typeof mod === 'string') {
```

Comment:
When can `mod` be a string? I think the processing above ensures it is always an array of entries like `{"name": "tedious", "minVersion": 18 }`.

Reply:
Leftover from a previous iteration where the `modules.findIndex` was not used